src/app/model/goal.ts

```diff
@@ -0,0 +1,10 @@
+export interface Goal {
```

Comment:
Good job! Probably add documentations for fields of which the meaning is not very clear from the name.

Reply:
ok